test-case/check-audio-equalizer.sh

```diff
@@ -52,95 +52,95 @@ func_test_eq()
         dloge "Equalizer setting failure with $conf"
         return 1
     fi
-    # test with aplay
-    dlogc "aplay -D$dev -f $fmt -c $channel -r $rate /dev/zero -d $duration"
-    aplay -D$dev -f $fmt -c $channel /dev/zero -d $duration
+
+    dlogc "$cmd -D $dev -f $fmt -c $channel -r $rate -d $duration $dummy_file"
+    $cmd -D $dev -f $fmt -c $channel -r $rate -d $duration $dummy_file
     if [ $? -ne 0 ]; then
         dloge "Equalizer test failure with $conf"
         return 1
     fi
     sleep 1
 }
 
+# this function performs IIR/FIR filter test
+# param1 must be must be iir or fir
+func_test_filter()
+{
+    local testfilter=$1
+    dlogi "1. Get amixer control id for $testfilter"
+    Filterid=`amixer -D hw:$sofcard controls | grep -i eq${testfilter}| head -1| sed 's/numid=\([0-9]*\),.*/\1/'`
+    if [ -z "$Filterid" ]; then
+        dloge "can't find $testfilter"
+        exit 1
```

Comment:
`fail` / `N/A`, I'm not sure if miss configure the behavior can accept  
couldn't : `failed`
just lost feature: `N/A`

Reply:
func_test_filter() will start when the topology support eq filter(s). Hence any failure while running the test should be reported as failure. In this case, we couldn't find alsactl id to test.